### ISSUE TITLE
Use extensible Circle CI configuration

### ISSUE THREAD

**Issue by gonzalo-bulnes** [freedomofpress/securedrop-updater#4]
## Description

I think it would be neat to setup the Circle CI configuration like the `securedrop-client` and `securedrop-export`. The main advantage of that most recent setup is that it allows to easily extend the scope of automated testing to new versions of Debian.

References:

- https://github.com/freedomofpress/securedrop-client/pull/1555
- https://github.com/freedomofpress/securedrop-export/pull/101

## Motivation

Mainly, I think it will make maintenance easier to have the Workstation components that are written in Python have a similar CI config structure, in the spirit of <missing link to an issue or PR that talked about standardizing repo structure - @eaon I'm sure you'll know which one I'm referring to!>.

**Comment by eaon** [comment]
I think that's a great idea, although since the target for this package would be dom0, we would probably want to test stuff with Fedora 32

**Comment by gonzalo-bulnes** [comment]
Good point! We can use a Fedora 32 image to start with and add the next in line when we know what the plan for Qubes OS 4.2 is.

**Comment by eaon** [comment]
Qubes OS R4.2 will ship with Fedora 37 as dom0, and given the current state of the repo I think it should be fairly straightforward to add Fedora 37 support.
